scripts/inventory.js

```diff
@@ -31,7 +31,13 @@ Game.prototype.removeFromInventory = function (itemName) {
 	this.drawInventory();
 
 	if (object.onDrop) {
-		object.onDrop();
+		this._setPlayerCodeRunning(true);
+		try {
+			object.onDrop();
```

Comment:
`validateCallback` can't be called here to avoid breaking Level 15, which calls `removeItem` in `startLevel`. `removeItem` ends up here, and calling `validateCallback` before `startLevel` returns is a bad idea.